src/playback-watcher.js

```diff
@@ -115,6 +115,24 @@ export default class PlaybackWatcher {
       this.tech_.on(['seeked', 'seeking'], loaderChecks[type].reset);
     });
 
+    // handle seeking into a gap
+    this.tech_.on('seeking', () => {
+      let seekTimeout;
+
+      const clear = () => {
+        this.tech_.off('seeking', clear);
+        this.tech_.off('seeked', clear);
+        window.clearTimeout(seekTimeout);
+      };
+
+      this.tech_.on('seeking', clear);
+      this.tech_.on('seeked', clear);
+
+      seekTimeout = window.setTimeout(() => {
+        clear();
+        this.techWaiting_(false);
```

Comment:
I wonder if there's a better way of whether we should consider seeking or not rather than overriding whether we're actually seeking. Technically we are in a seeking state, but maybe we can just have something like `ignoreSeeks`.